src/routes/api/installation/api-installation-get.ts

```diff
@@ -7,12 +7,13 @@ import { createAppClient } from "~/src/util/get-github-client-config";
 export const ApiInstallationGet = async (req: Request, res: Response): Promise<void> => {
 	const { installationId } = req.params;
 	const { client, jiraHost } = res.locals;
-	const gitHubAppClient = await createAppClient(req.log, jiraHost);
+	const { gitHubInstallationId } = req.body;
+	const gitHubAppClient = await createAppClient(gitHubInstallationId, req.log, jiraHost);
 
 	try {
-		const subscriptions = await Subscription.getAllForInstallation(Number(installationId));
+		const subscriptions = await Subscription.getAllForGitHubInstallationId(Number(installationId));
```

Comment:
Ignore above, I think the best way will be somehow validates all the params first, and here still just doing the `Number(installationId)` for sake of simply code.

Reply:
I'd prefer not to make that change as part of this PR as that is existing code. The only reason there is a change to this file is because I had to update the subscription method name.